akka-persistence-typed/src/main/scala/akka/persistence/typed/dstate/scaladsl/DurableStateBehavior.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed.dstate.scaladsl
+
+import akka.actor.typed.internal.BehaviorImpl.DeferredBehavior
+import akka.actor.typed.internal.LoggerClass
+import akka.actor.typed.{BackoffSupervisorStrategy, Signal}
+import akka.persistence.typed.PersistenceId
+import akka.persistence.typed.dstate.internal.DurableStateBehaviorImpl
+
+object DurableStateBehavior {
+
+  type CommandHandler[Command, State] = (State, Command) => Effect[State]
+
+  private val logPrefixSkipList = classOf[DurableStateBehavior[_, _]].getName :: Nil
+
+  def apply[Command, State](
+      persistenceId: PersistenceId,
```

Comment:
I wonder if it'd make sense to split this into a piece that's consistent over all entities of this type and an 'entityId' to identify each individual entity of this type?

Reply:
not sure if I understand your question but I guess it's related to the discussion here https://github.com/akka/akka-persistence-spanner/pull/103/files#r575046210

I think it is good to keep it closer to the existing persistence API, but also for the reasons I mentioned in https://github.com/akka/akka-persistence-spanner/pull/103/files#r575046210